Sensus.iOS/Concurrent/MainConcurrent.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright 2014 The Rector & Visitors of the University of Virginia
```

Comment:
Just an oversight. Apparently VS had simply removed from solution and not deleted. It's been deleted now.